v19.2/cockroachcloud-upgrade-policy.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cluster Upgrades
+summary: Learn about the CockroachCloud upgrade policy.
+toc: true
+build_for: [cockroachcloud]
+---
+
+This page describes the upgrade policy for CockroachCloud.
+
+CockroachCloud supports the latest major version of CockroachDB and the version immediately preceding it. Support for these versions includes minor version updates and security patches.
+
+## Minor Version Upgrades
+"Minor" versions (or "point" releases) are stable, backward-compatible improvements to the major versions of CockroachDB.
+
+CockroachCloud automatically upgrades all clusters to the latest supported minor version (for example, v19.2.1 → v19.2.2) within the 2 business days after a minor version release. Starting April, the [CockroachCloud Admin(s)](cockroachcloud-console-access-management.html#console-admin) will be notified before and after their clusters are upgraded to the latest minor version.
```

Comment:
Can we also leave out the part about email notifications? Again, this is the goal, but I don't want to commit to a deadline in case we need to cut scope.